src/main/java/org/embulk/input/zendesk/utils/ZendeskDateUtils.java

```diff
@@ -0,0 +1,48 @@
+package org.embulk.input.zendesk.utils;
+
+import org.embulk.spi.DataException;
+
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+import java.util.List;
+
+public class ZendeskDateUtils
+{
+    private static String supportedTimeFormat(final String value, final List<String> supportedFormats)
```

Comment:
Would be better to use `Optional` instead of empty string as non-parsable.

Reply:
fixed